lib/cyclopedio/syntax/name_decorator.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'stanford/all'
+require 'wiktionary/noun'
+
+module Cyclopedio
+  module Syntax
+
+    class NameDecorator < SimpleDelegator
```

Comment:
I think efficiency is not our main goal and method_missing performance is negligible. I can write performance tests about different decorator implementations ;) or tell me how you see it.